doc/rtd/tutorial/qemu.rst

```diff
@@ -80,6 +80,12 @@ server image using :command:`wget`:
 
     $ wget https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
 
+.. note::
+   Please be aware, the example is written for x86. It works on non-x86 but
```

Comment:
Same comments as above, however the this is a tutorial, not an example, so suggest a one-word change over the previous suggestion: 

> This tutorial uses emulated CPU instructions on non-x86 hosts, so it may be slow. To make it faster on non-x86 architectures, one can change the image type and qemu-system- command name to match the architecture of your host machine.